pkg/scheduler/apis/config/v1beta3/default_plugins_test.go

```diff
@@ -97,6 +97,38 @@ func TestApplyFeatureGates(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
This test should go away in v1beta3 soon.
We decided to check the feature gate in the plugin instead instead of changing the configuration.